edkrepo/commands/arguments/sync_args.py

```diff
@@ -14,4 +14,5 @@
 COMMAND_DESCRIPTION = 'Updates the local copy of the current combination\'s target branches by pulling the latest changes from the server. Does not update local branches.'
 FETCH_HELP = 'Performs a fetch only sync, no changes will be made to the local workspace.'
 UPDATE_LOCAL_MANIFEST_HELP = 'Updates the local copy of the project manifest file prior to performing sync operations.'
-OVERRIDE_HELP = 'Ignore warnings and proceed with sync operations.'
\ No newline at end of file
+OVERRIDE_HELP = 'Ignore warnings and proceed with sync operations.'
+TAG_HELP = 'Enables tags to be pulled and TC_* tags to be removed. Using this flag will result in a significantly longer sync time.'

```

Comment:
"and TC_* tags to be removed"
The code below does "git fetch --tags"; I don't think that will remove any tags unless --force is used.  Update TAG_HELP string?

Reply:
Help string updated